src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,319 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Duke {
+    public static void printLines() {
+        System.out.println("_________________________________________________________________________");
+    }
+
+    public static void printLinesWithText(String response) {
+        //Prints horizontal lines in between response
+        printLines();
+        System.out.println(response);
+        printLines();
+        System.out.print(System.lineSeparator());
+    }
+
+    private static void printWelcomeMessage() {
+        String logo = "  _  _     ___    _____   _____    ___\n"
+                + " | \\| |   /   \\  |_   _| |_   _|  / _ \\\n"
+                + " | .` |   | - |    | |     | |   | (_) |\n"
+                + " |_|\\_|   |_|_|   _|_|_   _|_|_   \\___/\n"
+                + "_|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"|\n"
+                + "\"`-0-0-'\"`-0-0-'\"`-0-0-'\"`-0-0-'\"`-0-0-'\n";
+        System.out.println(logo);
+        printLinesWithText("Top of the mornin' to ya! Name's Natto" + System.lineSeparator()
+                + "Need a pint, two, or somethin' else?");
+    }
+
+    private static void printGoodbyeMessage() {
+        //Prints goodbye string after breaking out of loop
+        printLinesWithText("What, so everyone's supposed to sleep every night now?" + System.lineSeparator()
+                + "You realize that nighttime makes up half of all time?");
+    }
+
+    private static void printAddTaskSuccess(Task task) {
+        printLinesWithText("Added that one to the list: " + task);
+    }
+
+    private static void printDeleteTaskSuccess(Task task) {
+        printLines();
+        System.out.println("You've removed the task but not my disappointment:" + System.lineSeparator()
+                + task.toString()  + "  -------->" + "deleted");
+        System.out.println("You've got " + Task.getTaskCount() + " more tasks to delete instead of complete");
+        printLines();
+    }
+
+    private static String printTodoFormat() {
```

Comment:
The name of this method is not very clear as the method does not actually do any printing.